docs/pages/concepts/how-liveblocks-works.mdx

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: "How Liveblocks works"
+  parentTitle: "Concepts"
+  description:
+    "Liveblocks is a real-time collaboration infrastructure for building
+    performant collaborative experiences."
+---
+
+Liveblocks is a real-time collaboration infrastructure for building performant
+collaborative experiences. Liveblocks is built around the following core
+concepts: primitives, rooms, integrations, and platform.
+
+## Primitives
+
+Liveblocks is a fully integrated solution built around core primitives to embed
+collaborative experiences: [Presence](/docs/primitives/presence),
+[Broadcast](/docs/primitives/broadcast), and
+[Storage](/docs/primitives/storage).
+
+<Figure highlight>
+  <Image
+    src="/images/docs/concepts/primitives.png"
+    alt="How Liveblocks works - Primitives"
+    layout="responsive"
+    width={768}
+    height={440}
+    quality={90}
+  />
+</Figure>
+
+## Rooms
+
+A room is the digital space in which people collaborate. Primitives are
+associated with dedicated rooms.
```

Comment:
```suggestion
Rooms are digital spaces in which people collaborate. Primitives are
used within separate rooms.
```
I think this was too vague before. Better?

Reply:
I made some changes in between so this suggestion is outdated. I think what I have now is clearer, what do you think?

Thanks!